homeassistant/components/vicare/water_heater.py

```diff
@@ -94,6 +97,14 @@ async def async_setup_entry(
 
     async_add_entities(entities)
 
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
```

Comment:
I suggest you create a button entity instead and a binary sensor that represents the active state.

Reply:
I force-pushed to start over.